api/src/pcapi/core/fraud/factories.py

```diff
@@ -235,3 +235,13 @@ class Meta:
         model = models.BlacklistedDomainName
 
     domain = factory.Faker("domain_name")
+
+
+class ProductWhitelistFactory(testing.BaseFactory):
+    class Meta:
+        model = models.ProductWhitelist
+
+    comment = factory.Sequence("OK {} !".format)
+    title = factory.Sequence("Ducobu #{} !".format)
+    ean = factory.fuzzy.FuzzyText(length=13)
+    dateCreated = factory.LazyFunction(lambda: datetime.now(tz=pytz.utc))  # pylint: disable=datetime-now
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
```suggestion
    dateCreated = factory.LazyFunction(lambda: datetime.utcnow())
```